components/rmrk/Gallery/Search/SearchBar.vue

```diff
@@ -58,14 +57,14 @@
         </b-field>
         <slot />
       </div>
-
-      <transition name="fade">
-        <div v-if="isVisible" class="columns">
-          <Sort class="column is-4 mb-0" :value="sortBy" @input="updateSortBy" />
-          <BasicSwitch class="column is-4" v-model="vListed" :label="!replaceBuyNowWithYolo ? 'sort.listed' : 'YOLO'" size="is-medium" labelColor="is-success" />
-        </div>
-      </transition>
-
+      <div class="filter-container" :class="{open: isVisible}">
+        <transition name="fade">
+          <div v-if="isVisible" class="columns">
+            <Sort class="column is-4 mb-0" :value="sortBy" @input="updateSortBy" />
+            <BasicSwitch class="column is-4" v-model="vListed" :label="!replaceBuyNowWithYolo ? 'sort.listed' : 'YOLO'" size="is-medium" labelColor="is-success" />
+          </div>
+        </transition>
+      </div>
```

Comment:
Why do we need a container that wrapp transition here? I think this could work without.
or I would move `v-if` to filter-container

Reply:
tried both, but the height transition doesn't work properly if the element is added to dom via v-if
https://user-images.githubusercontent.com/17953978/148030726-c7012b91-6dbc-4001-84eb-af93074f384e.mov

but you are right about the whitespace... i'll fix that later today 😅

Reply:
@roiLeo nevermind, i just found a much more elegant solution using Buefy's collapse component with a custom animation 😸 
and i also removed the weird margin from pagination -> it now looks much more symmetrical, hope you are ok with that

https://user-images.githubusercontent.com/17953978/148173227-e0a40efb-dfa0-4497-8986-1aec69c45c42.mov

Reply:
mobile looks slick too now 💯 
<img width="410" alt="Screenshot 2022-01-05 at 07 53 53" src="https://user-images.githubusercontent.com/17953978/148173456-be33f859-61f4-4cc0-a584-818da80a0c79.png">